docs/extensibility/slot-fills.md

```diff
@@ -0,0 +1,50 @@
+# Slots and Fills
+
+## The problem
+You added custom data to the [Store API](./extend-rest-api-add-data.md). You changed several strings using [Checkout filters](./available-filters.md). Now you want to render your own components in specific places in the Cart and Checkout.
+
+## Solution
+
+Slots and Fills add the possibility to render your own HTML in pre-defined places in the Cart and Checkout. Your component will get access to contextual data and will get re-rendered when needed.
+
+A Slot is a place in the Cart and Checkout that can render an indefinite number of external components.
+
+A Fill is the component provided by third-party developers to render inside a Slot.
+
+Slots and Fills uses WordPress's API, [so you can expand more on its API there.](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/slot-fill).
+
+## Basic Usage
+
+`ExperimentalOrderMeta` is a fill that will render in a slot below the order summary section in the Cart and Checkout blocks.
```

Comment:
It might be worth linking over to the doc where you explain more about `ExperimentalOrderMeta` here.